libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -287,16 +348,15 @@ void AP_Frsky_Telem::send_SPort(void)
                         case 1:
                             send_sport_frame(SPORT_DATA_FRAME, DATA_ID_VFAS, (uint16_t)roundf(_battery.voltage() * 10.0f)); // send battery voltage
                             break;
-                        case 2:
```

Comment:
Whitespace changes not associated with the PR.

I still have to stare at them every time I go through the code.

Maybe break them out into a separate PR?